libs/coin-modules/coin-evm/src/logic.ts

```diff
@@ -173,15 +172,6 @@ const CALHashByChainIdMap = new Map<CryptoCurrency, string>();
  * Getter for the CAL content hash
  */
 export const getCALHash = (currency: CryptoCurrency): string => {
-  if (!CALHashByChainIdMap.has(currency)) {
-    CALHashByChainIdMap.set(
-      currency,
-      tokensHashesByChainId[
-        (currency?.ethereumLikeInfo?.chainId || "") as keyof typeof tokensHashesByChainId
-      ],
-    );
-  }
-
   return CALHashByChainIdMap.get(currency) || "";
```

Comment:
CryptoCurrency is a map from CryptoCurrency(object) -> hash(string)
It would be better if it is a map from CryptoCurrency.id(string) -> hash(string)

Reply:
I'm not sure it's really changing anything here, Map have been designed to have any content type as keys, so I'm not sure it would improve something here 🤔